packages/algob/src/lib/account.ts

```diff
@@ -98,7 +98,7 @@ export function loadAccountsFromEnv (): Account[] {
 }
 
 // returns multisignature account address
-export function createMsigAddress (version: number, threshold: number, accountList: Account[]): any {
+export function createMsigAddress (version: number, threshold: number, accountList: string[]): any {
```

Comment:
Could you define a proper type instead of using `any`?

Reply:
ok